### DISCUSSION TITLE
MacOS使用程序时发生Log类的"DS_?"不符合时间格式的报错

### DISCUSSION THREAD

**Discussion by Sy1vian** [MashiroSA/sekai-voice-fetch#9] [Q&A] [answered]
## 测试环境
- Python 3.10
- Pycharm
- Venv
- MacOS

## 发生的问题
MacOS使用程序时发生Log类的"DS_?"不符合时间格式的异常报错。

## 日志
```

```

**Comment by chitsanfei** [comment, marked as answer]
这是由于MacOS会自动生成`.DS_Store`，以优化Finder的功能，若出现这种问题，请删除`Logger/logs`文件夹或文件夹中的`.DS_Store`以修复这个问题。
